### DISCUSSION TITLE
Was running automated tests but they seem to be failing now

### DISCUSSION THREAD

**Discussion by kevkevinpal** [sequelize/sequelize#14614] [Help & Questions] [answered]
## Issue Creation Checklist

- [X] I have read the [contribution guidelines](https://github.com/sequelize/sequelize/blob/main/CONTRIBUTING.md)

## Issue Description

Hey so I was running some automated tests and they suddenly started to fail with a typescript error and it seems to be an issue coming from sequelize but I'm not totally sure why here is a log of the test. It's failing on the `npm run build` command

https://github.com/stakwork/sphinx-relay/runs/6819507922?check_suite_focus=true

### Additional context
https://github.com/stakwork/sphinx-relay/runs/6819507922?check_suite_focus=true

### Is this issue dialect-specific?

- [ ] No. This issue is relevant to Sequelize as a whole.
- [ ] Yes. This issue only applies to the following dialect(s): XXX, YYY, ZZZ
- [X] I don't know.

### Would you be willing to resolve this issue by submitting a Pull Request?

- [ ] Yes, I have the time and I know how to start.
- [ ] Yes, I have the time but I don't know how to start, I would need guidance.
- [ ] No, I don't have the time, although I believe I could do it if I had the time...
- [X] No, I don't have the time and I wouldn't even know how to start.

**Comment by ephys** [comment, marked as answer]
[This PR](https://github.com/stakwork/sphinx-relay/pull/380) installed Sequelize ^5.19.3, but the lockfile still used 5.19.3. If you use `npm i` instead of `npm ci`, it's possible npm is going to install Sequelize 5.22.5 instead of 5.19.3. It's possible stricter typings were introduced between these two versions.

Note that Sequelize 5 is not maintained anymore: https://sequelize.org/releases/

**Reply by kevkevinpal** [reply]
So it seem the issue was the node version we were running, not sure why the failures started to happen all of a sudden but they seem to be resolved now. Thanks for taking a look :)

https://github.com/stakwork/sphinx-relay/pull/385
